spec/lib/vanagon/platform_spec.rb

```diff
@@ -4,22 +4,31 @@
   let(:platforms) do
     [
       {
-        :name         => "debian-6-i386",
-        :os_name      => "debian",
-        :os_version   => "6",
-        :architecture => "i386",
+        :name                   => "debian-6-i386",
+        :os_name                => "debian",
+        :os_version             => "6",
+        :architecture           => "i386",
+        :output_dir             => "debian/6/i386",
+        :output_dir_with_target => "debian/6/thing/i386",
```

Comment:
Yeah, so the deal here is actually that we are going to have to add an override to each platform config for debian platforms in the projects themselves to conform to stuff like `deb/lucid/thing` . The intent of this PR is not to change any of the output paths, but instead move the functionality of specialized output pathing out of vanagon and in to the projects.